NativeApp/src/Android/AndroidAppBase.cpp

```diff
@@ -113,27 +127,91 @@ int32_t AndroidAppBase::HandleInput(android_app* app, AInputEvent* event)
 //
 // Process the next main command.
 //
-void AndroidAppBase::HandleCmd(struct android_app* app, int32_t cmd)
+void AndroidAppBase::HandleCmd(struct android_app* android_app, int32_t cmd)
 {
-    AndroidAppBase* eng = (AndroidAppBase*)app->userData;
+    AndroidAppBase* eng = (AndroidAppBase*)android_app->userData;
     switch (cmd)
     {
-        case APP_CMD_SAVE_STATE:
+        case APP_CMD_INPUT_CHANGED:
+#if(LOG_NATIVE_APPLICATION)
+            LOG_ERROR_MESSAGE("APP_CMD_INPUT_CHANGED");
```

Comment:
Why is it an error?

Reply:
to make it stand out in the log as it is important for seeing what is going on with the state of the native app internally, since errors have red text in logcat